### ISSUE TITLE
Unable to install from NPM

### ISSUE THREAD

**Issue by misscs** [mahnunchik/gulp-responsive#14]
Currently getting the following error when I try to run ``npm install gulp-responsive`. 

```
fetch failed https://registry.npmjs.org/gulp-responsive/-/gulp-responsive-1.1.0.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
```

ps. Congrats on the new release

**Comment by misscs** [comment]
Works if i point npm to the repo

**Comment by mahnunchik** [comment]
Hi @misscs 
Thanks!

I've faced with the same error https://github.com/npm/npm/issues/8354

I will try to re-publish this release with the version number 1.1.1.

**Comment by mahnunchik** [comment]
I've published version 1.1.1. Installation error seems to be resolved.

But version 1.1.0 still can not be installed. I think it is the NPM bug.

**Comment by jonathandion** [comment]
Hey @mahnunchik I still can't install it with npm.

**Comment by mahnunchik** [comment]
Hi @jonathandion 

Could you please provide what npm say.

Which version of `gulp-responsive` did you install?

**Comment by mahnunchik** [comment]
> problem was that the version of node that published it was 0.11.16, which is broken and can't publish valid npm packages (a broken crypto.Hash implementation).

https://github.com/npm/npm/issues/8354#issuecomment-119432583
